.env.example

```
@@ -1,2 +1,4 @@
 CONFIG_SERVER_USERNAME=
 CONFIG_SERVER_PASSWORD=
+GITHUB_USER=
+GITHUB_TOKEN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_TOKEN key should go before the GITHUB_USER key

(UnorderedKey)